constraint/pkg/apis/templates/v1beta1/defaults.go

```diff
@@ -0,0 +1,28 @@
+package v1beta1
+
+import (
+	"github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates"
+	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema/defaulting"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const version = "v1beta1"
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	return RegisterDefaults(scheme)
+}
+
+func SetDefaults_ConstraintTemplate(obj *ConstraintTemplate) { //nolint:golint
+	// turn the CT into an unstructured
+	un, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+	if err != nil {
+		panic("Failed to convert v1 ConstraintTemplate to Unstructured")
```

Comment:
Is there a way to let these return error, or does that mess up the code generation logic?

Reply:
Unfortunately the expected function signature does not allow for returning an error

Reply:
Ah, that's disappointing.